spa/src/api/apps/index.ts

```diff
@@ -0,0 +1,9 @@
+import { GetGitHubAppsUrlResponse } from "../../rest-interfaces/oauth-types";
+import AxiosInstance from "../axiosInstance";
+import { AxiosResponse } from "axios";
+
+const GitHubApps = {
+	getAppNewInstallationUrl: async (): Promise<AxiosResponse<GetGitHubAppsUrlResponse>> => AxiosInstance.get("/rest/app/cloud/installation/new"),
```

Comment:
Do we need the `async` here? The async is already implied by the Promise definition.

Reply:
Probably don't, can fix that.

Reply:
fixed.